server/impl/src/main/java/com/walmartlabs/concord/server/org/project/RepositoryDao.java

```diff
@@ -38,12 +39,15 @@
 public class RepositoryDao extends AbstractDao {
 
     private final ConcordObjectMapper objectMapper;
+    private final TriggersDao triggersDao;
 
     @Inject
     public RepositoryDao(@MainDB Configuration cfg,
-                         ConcordObjectMapper objectMapper) {
+                         ConcordObjectMapper objectMapper,
```

Comment:
I think it might be cleaner to make a small internal Dao class in GithubTriggerV2Processor and "compose" RepositoryDao and TriggersDao there instead of making a dependency here.